inventories/default/group_vars/all.yaml.template

```
@@ -11,7 +11,7 @@ env:
 # Section 2 - LPAR(s)
   z:
     high_availability: False
-    ip_forward: #X
+    ip_forward: 0
```

```suggestion
    ip_forward: "{{ 1 if network_mode | upper == 'NAT' else 0 }}"
```
What do you think about changing to this?

---

One less variable to forget to change when using NAT.

---

This make sense ...